HunterPie.Core/Native/IPC/Handlers/Internal/Initialiaze/Models/RequestIPCInitializationMessage.cs

```diff
@@ -9,7 +10,28 @@ public struct RequestIPCInitializationMessage
     {
         public IPCMessage Header;
 
+        public IPCInitializationHostType HostType;
+
+        private readonly int Reserved1;
```

Comment:
This field name is very counterintuitive and is also not needed

Reply:
Thus I'll remove it